src/current/v24.1/molt-fetch.md

```diff
@@ -275,23 +309,106 @@ To drop existing tables and create new tables before loading the data, use `'dro
 --table-handling 'drop-on-target-and-recreate'
 ~~~
 
-With each option, MOLT Fetch creates a new CockroachDB table to load the source data if one does not exist.
+When using the `'drop-on-target-and-recreate'` option, MOLT Fetch creates a new CockroachDB table to load the source data if one does not already exist. To guide schema creation, you can [explicitly map source types to CockroachDB types](#type-mapping).
+
+#### Type mapping
+
+If [`'drop-on-target-and-recreate'`](#target-table-handling) is set, MOLT Fetch automatically creates a CockroachDB schema that is compatible with the source data. The column types are determined as follows:
+
+- All PostgreSQL types are mapped to identical CockroachDB [types]({% link {{ page.version.version }}/data-types.md %}).
```

Comment:
```suggestion
- A PostgreSQL type is mapped to a CockroachDB [type]({% link {{ page.version.version }}/data-types.md %}) with the same type oid.
```

Reply:
Yeah i feel torn here as well. In contrast to `same` or `identical`, maybe `corresponding` would be better?

The tricky thing with PG is that [CRDB](https://www.cockroachlabs.com/docs/stable/data-types) only support a subset of [PG types](https://www.postgresql.org/docs/current/datatype.html). For example, CRDB doesn't have a `money` type from PG.  But for those overlapped by PG and CRDB, most of them are 1-1. 

MySQL is a bit tricky as it can be many-to-one, which makes `same` or `identical` less accurate.
Is it ok if we say that a PG type is mapped to the corresponding CockroachDB type based on the oid, if applicable. For MySQL, since we support a much smaller set of mysql types, I think it's ok for us to exhaust all mapping rules. can we still keep those entries even they are the same for mysql and cockroachdb?

Reply:
I see! OK, I made the table complete again.